src/declarative/components/input/PIRMotionSensor.tsx

```diff
@@ -0,0 +1,51 @@
+import type { SerialPort } from 'serialport'
+import React, { createContext } from 'react'
+import { board } from '../../../procedure/utils/board'
+import { attachInput } from '../../../procedure/examples/input/uniqueDevice/input'
+
+export const PIRMotionSensorContext = createContext<SerialPort | null>(null)
+
+type PIRMotionSensorProps = {
+  pin: number
+  triggered?: () => void
+  untriggered?: () => void
+  children: React.ReactNode
+}
+
+export const PIRMotionSensor: React.FC<PIRMotionSensorProps> = ({
+  pin,
+  triggered,
+  untriggered,
+  children,
+}) => {
```

Comment:
add optional property "delayTime" that can indicate on code for device specification.